example/alipay/config_example_category.yaml

```diff
@@ -0,0 +1,73 @@
+defaultMinusAccount: Assets:FIXME

```

Comment:
I think you should add:
- example `alipay` records to `example/alipay/example-alipay-records.scv`.
- example `alipay` config to `example/alipay/config.yaml`.
- example `alipay` outputs to `example/alipay/example-alipay-output.beancount` (for further integration test like `wechat` provider).

Don't need to create new files.